src/sys/socket/mod.rs

```diff
@@ -380,7 +388,7 @@ pub enum ControlMessage<'a> {
 pub struct UnknownCmsg<'a>(&'a cmsghdr, &'a [u8]);
 
 fn cmsg_align(len: usize) -> usize {
```

Comment:
I'd suggest we add an `#[inline]` here as it's a small convenience function.

Reply:
Could you also add a comment here summarizing this function? I'm having trouble following this code and having a comment for this helper function would be nice.